service-account-generator/README.md

```diff
@@ -0,0 +1,20 @@
+# Generating service accounts for the CICD pipeline
+
+The github actions need service accounts to run. The script `generate_sa.sh` will create a service acount for the prod environment of a given openshift project and give that account the roles in the dev, test, and prod environments for deploying the keycloak site.
```

Comment:
Probably need an update to say Realm Registry site??

Reply:
good catch